src/Components/Components/src/HotReload/HotReloadContext.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Components.HotReload
+{
+    /// <summary>
+    /// A context that indicates when a component is being rendered after a hot reload is applied to the application.
```

Comment:
```suggestion
    /// A context that indicates when a component is being rendered because of a hot reload operation.
```

I'm being pendantic, but clarifying that it's not *all* renders that happen after any preceding hot reload.